charts/cluster-autoscaler/values.yaml

```diff
@@ -298,6 +298,19 @@ serviceMonitor:
   # serviceMonitor.path -- The path to scrape for metrics; autoscaler exposes `/metrics` (this is standard)
   path: /metrics
 
+## Custom PrometheusRule to be defined
+## The value is evaluated as a template, so, for example, the value can depend on .Release or .Chart
+## ref: https://github.com/coreos/prometheus-operator#customresourcedefinitions
+prometheusRule:
+  # prometheusRule.enabled -- If true, creates a Prometheus Operator PrometheusRule.
+  enabled: false
+  # prometheusRule.additionalLabels -- Additional labels to be set in metadata.
+  additionalLabels: {}
+  # prometheusRule.namespace -- Namespace which Prometheus is running in.
+  namespace: monitoring
+  # prometheusRule.rules -- Rules spec template.
```

Comment:
Is it worth linking to the API spec for the rule object here? I.e. https://github.com/prometheus-operator/prometheus-operator/blob/master/Documentation/api.md#rule

Reply:
Added the reference doc link. I'm thinking about adding an example too, what do you think ?